src/cp/cp_control_point.c

```diff
@@ -169,116 +171,81 @@ __wt_conn_control_point_init_all(WT_SESSION_IMPL *session)
      */
     /* From examples/ex_control_points.c */
     control_points[WT_CONN_CONTROL_POINT_ID_MAIN_START_PRINTING].init =
-      __wt_control_point_pair_init_always_wait_for_trigger;
-    control_points[WT_CONN_CONTROL_POINT_ID_MAIN_START_PRINTING].pred = NULL; /* Always */
+      __wt_control_point_pair_init_pred_wait_for_trigger;
```

Comment:
Rename to __wt_control_point_action_init_wait_for_trigger.

Reply:
IMO, i prefer the `__wt_control_point_pair_init_pred_wait_for_trigger` name here, and `__wt_control_point_action_init_wait_for_trigger` already exists as a function name.